js/ui/pivot_grid/ui.pivot_grid.area_item.js

```diff
@@ -387,7 +387,8 @@ exports.AreaItem = Class.inherit({
             colgroupElementHTML += '<col style="width: ' + columnWidth[i] + 'px">';
         }
         this._colgroupElement.html(colgroupElementHTML);
-        this._tableWidth = totalWidth - this._groupWidth > 0.01 ? Math.ceil(totalWidth) : totalWidth;
+        this._tableWidth = totalWidth;
```

Comment:
I suggest to use the `this._tableWidth`  instead of the `totalWidth` and remove the `totalWidth` varaible.

Reply:
Fixed. I have replaced it by columnWidth.reduce((sum, width) => sum + width, 0)